rocAL/source/api/rocal_api_meta_data.cpp

```diff
@@ -24,9 +24,9 @@ THE SOFTWARE.
 // Created by mvx on 3/31/20.
 //
 
-#include "commons.h"
-#include "context.h"
-#include "rocal_api.h"
```

Comment:
@LakshmiKumar23 we may have to keep this "rocal_api.h" as this going to be public. Lets discuss.

Reply:
Oh yes, i un-did this in most places. I will keep this as rocal_api.h